Rocky.class.nut

```diff
@@ -53,7 +83,23 @@ class Rocky {
     // Requests
     function on(verb, signature, callback, timeout=null) {
```

Comment:
function needs header e.g

    // Description of function
    //
    // Parameters:
    //      x              description of x
    //      cb                  callback function to be called when response received from aws